src/screens/FeedScreen/ContentPoll.js

```diff
@@ -265,20 +266,20 @@ const stylesComponent = (pollLength) =>
     },
     pollOptionItemText: {
       flex: 1,
-      color: COLORS.black,
+      color: colors.black,
       fontFamily: fonts.inter[400]
     },
     totalpolltext: {
       fontFamily: fonts.inter[400],
       fontSize: normalizeFontSize(12),
       lineHeight: 16,
-      color: COLORS.blackgrey
+      color: colors.blackgrey
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color property has been updated from `COLORS.blackgrey` to `colors.blackgrey`. Make sure that `colors.blackgrey` is defined and is the intended color.
```diff
- color: COLORS.blackgrey
+ color: colors.blackgrey
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->